configs/sites/hera/compilers.yaml

```diff
@@ -11,11 +11,7 @@ compilers:
     target: any
     modules:
     - intel/2022.1.2
-    environment:
-      prepend_path:
-        PATH: '/apps/gnu/gcc-9.2.0/bin'
-        LD_LIBRARY_PATH: '/apps/gnu/gcc-9.2.0/lib64'
-        CPATH: '/apps/gnu/gcc-9.2.0/include'
+    environment: {}
```

Comment:
Please do not remove this, it will break JEDI code that needs a newer gcc/g+++ backend than the default gcc-4.8.5 (for C++-14).

Reply:
Okay. I think I ran into an issue with a missing library (or wrong version thereof) at runtime at one point-- Would it make sense to prepend /apps/gnu/gcc-9.2.0/lib64 to LD_LIBRARY_PATH in the meta modulefile just to be safe?

Reply:
That's already in there, automatically:
```
[Dom.Heinzeller@hfe09 ~]$ module purge
[Dom.Heinzeller@hfe09 ~]$ module use /scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles
[Dom.Heinzeller@hfe09 ~]$ module load miniconda/3.9.12
[Dom.Heinzeller@hfe09 ~]$ module load ecflow/5.5.3
ECF_HOST = hecflow01, ECF_PORT = 21751
[Dom.Heinzeller@hfe09 ~]$ module use /scratch1/NCEPDEV/global/spack-stack/spack-stack-v1/envs/skylab-2.0.0-intel-2021.5.0/install/modulefiles/Core
[Dom.Heinzeller@hfe09 ~]$ module show stack-intel/2021.5.0
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   /scratch1/NCEPDEV/global/spack-stack/spack-stack-v1/envs/skylab-2.0.0-intel-2021.5.0/install/modulefiles/Core/stack-intel/2021.5.0.lua:
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
help([[]])
family("MetaCompiler")
conflict("stack-apple-clang")
conflict("stack-clang")
conflict("stack-gnu")
conflict("stack-intel")
conflict("stack-intel-oneapi-compilers")
load("intel/2022.1.2")
prereq("intel/2022.1.2")
prepend_path("MODULEPATH","/scratch1/NCEPDEV/global/spack-stack/spack-stack-v1/envs/skylab-2.0.0-intel-2021.5.0/install/modulefiles/intel/2021.5.0")
setenv("F77","/apps/oneapi/compiler/2022.0.2/linux/bin/intel64/ifort")
setenv("FC","/apps/oneapi/compiler/2022.0.2/linux/bin/intel64/ifort")
setenv("CC","/apps/oneapi/compiler/2022.0.2/linux/bin/intel64/icc")
setenv("CXX","/apps/oneapi/compiler/2022.0.2/linux/bin/intel64/icpc")
setenv("SERIAL_F77","/apps/oneapi/compiler/2022.0.2/linux/bin/intel64/ifort")
setenv("SERIAL_FC","/apps/oneapi/compiler/2022.0.2/linux/bin/intel64/ifort")
setenv("SERIAL_CC","/apps/oneapi/compiler/2022.0.2/linux/bin/intel64/icc")
setenv("SERIAL_CXX","/apps/oneapi/compiler/2022.0.2/linux/bin/intel64/icpc")
prepend_path("PATH","/apps/gnu/gcc-9.2.0/bin")
prepend_path("LD_LIBRARY_PATH","/apps/gnu/gcc-9.2.0/lib64")
prepend_path("CPATH","/apps/gnu/gcc-9.2.0/include")
whatis("Name: stack-intel")
whatis("Version: 2021.5.0")
whatis("Category: compiler")
whatis("Description: stack-intel compiler family and module access")
```